includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2609,14 +2609,10 @@ protected function get_user_formatted_tokens_array( $user_id ) {
 			]
 		);
 		return array_map(
-			static function ( WC_Payment_Token_CC $token ): array {
+			static function ( WC_Payment_Token $token ): array {
```

Comment:
Reason: The return type of `WC_Payment_Tokens::get_tokens` is `WC_Payment_Token[]`.

`$tokens` here are the inheritances of `WC_Payment_Token` including both `WC_Payment_Token_CC` and `WC_Payment_Token_WCPay_SEPA`.